### ISSUE TITLE
Text in place of Disqus

### ISSUE THREAD

**Issue by SpookyYomo** [nusmodifications/nusmods#3782]
### Is your feature request related to a problem? Please describe.

With reference to https://github.com/nusmodifications/nusmods/issues/3766#issuecomment-2295340941, Disqus comments do not show up for various reasons, such as Privacy Protectors, particularly so on the likes of privacy-focused browsers, such as Brave (not tested), Hardened Firefox (see: ArkenFox) etc.

### Describe the solution you'd like

Just as there are websites that still have a box around the expected location of an advertisement begging for you to disable your ad-block, we could have a placeholder suggested box that suggests Disqus has failed to load due to the user's browser. If the user-agent string is Firefox, as in the referenced comment, suggest to disable Enhanced Tracking Protection and reload the page. (Firefox extensions, with the likes of DecentralEyes and PrivacyBadger should be left to the user to decipher.)

### Describe alternatives you've considered

We hope the user doesn't realise Disqus is a thing.

**Comment by ravern** [comment]
@SpookyYomo shall we scope this issue to only solving for Firefox right now (via user-agent string as mentioned)?

We could figure out expanding the detection of ad-blocking in general to future issues.

**Comment by joshcangit** [comment]
If the issue is with Firefox's Enhanced Tracking Protection, then just add **nusmod.com** to the Exceptions and save it.
That way, the whole site including the comment section would be unblocked.

**Comment by ravern** [comment]
> If the issue is with Firefox's **Enhanced Tracking Protection**, then just add **nusmod.com** to **Manage Exceptions...** and save it. That way, the whole site including the comment section would be unblocked.

Yeah, that's the point of this issue, to have some kind of infobox on NUSMods to inform the user of this solution.

**Comment by SpookyYomo** [comment]
> @SpookyYomo shall we scope this issue to only solving for Firefox right now (via user-agent string as mentioned)?
> 
> We could figure out expanding the detection of ad-blocking in general to future issues.

Hi @ravern, I am not privy to the difficulties of frontend myself, but I assumed drawing a box in place of Disqus with an "Oopsie, we couldn't load the comments from Disqus, please check your browser!" would have been equally difficult across browsers, but for firefox you'd just have an if conditional to change the displayed text to then add "Do you have Firefox Enhanced Tracking Protection enabled?" or something similar.
My issue filed was just a suggestion to provide end user indication and not necessary propose the exact means by which this end user indicator should have been, please do what makes most sense for as a dev and for the project, freely bending the spirit of my little suggestion :)

**Comment by SpookyYomo** [comment]
<img width="1117" height="601" alt="Image" src="https://github.com/user-attachments/assets/c36c2d51-e5df-4587-b9f2-e8aa8f628175" />
Screenshot as of Firefox 146.


If someone else can ascertain that this (dialogue showing that Disqus
is blocked) is a Firefox default now, perhaps we can close the issue.
(Disqus currently loads in Brave on my setup).

**Comment by ravern** [comment]
Hey, we're doing some clean ups over the summer. Closing this for now because this issue is stale. Feel free to re-open this if you think otherwise!
